src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -119,11 +123,20 @@
   [a b]
   (or (= a b) (some #{:any} [a b])))
 
+(defn- query-params-match?
```

Comment:
Thanks for the explanation, that makes sense. One could argue that you can test through the public interface of the module and your tests shouldn't have any knowledge of the gory internal details (hence `defn-` usage), but I see the point.